src/main/java/racingcar/controller/GameManager.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.domain.Cars;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class GameManager {
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public GameManager(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        Cars cars = new Cars(inputView.readCarNames());
+        int count = inputView.readTryCount();
```

Comment:
retry를 사용하고 싶을 때는, 아래와 같이 사용한다.

```suggestion
Cars cars = retry(inputView::readCarNames);
int count = retry(inputView::readTryCount);
```